ui/src/app/cohort-search/attributes-page-v2/attributes-page-v2.component.tsx

```diff
@@ -126,23 +126,38 @@ const styles = reactStyles({
   },
 });
 
-export const CalculateFooter = (props) => {
-  return <FlexRowWrap style={styles.countPreview}>
-     <div style={styles.resultsContainer}>
-     <Button id='attributes-calculate'
-       type='secondaryLight'
-    disabled={props.disabled}
-    style={{...styles.calculateButton, ...(props.disabled ? {borderColor: colorWithWhiteness(colors.dark, 0.6)} : {})}}
-    onClick={() => props.calculate()}>
-    {props.calculating && <Spinner size={16} style={styles.spinner}/>} Calculate
-    </Button>
-    </div>
-    <div style={styles.resultsContainer}>
-      <div style={{fontWeight: 600}}>Number of Participants:
-        <span> {props.count === null ? '--' : props.count.toLocaleString()} </span>
+export const CalculateFooter = ({addButtonText, addFn, backFn, calculateFn, calculating, count, disableAdd, disableCalculate}) => {
+  return <div style={{background: colorWithWhiteness(colors.primary, .87), bottom: 0, position: 'sticky'}}>
+    <FlexRowWrap style={styles.countPreview}>
+      <div style={styles.resultsContainer}>
+        <Button id='attributes-calculate'
+          type='secondaryLight'
+          disabled={disableCalculate}
+          style={{...styles.calculateButton, ...(disableCalculate ? {borderColor: colorWithWhiteness(colors.dark, 0.6)} : {})}}
```

Comment:
nit: can we re-write style condition as disableCalculate ? {....styles.calculateButton, borderColor: colorWithWhiteness(colors.dark, 0.6)} : styles.calculatebutton